### PR TITLE
Add squeeze to distributions

### PR DESCRIPTION
**Pull request by mfvescovi**
This change is intended to allow test builds for inclusion in Squeeze LTS.

### CODE REVIEW

File: etc/debomatic/distributions [hunk 1]
```diff
@@ -38,6 +38,12 @@ extramirrors: deb http://security.debian.org/ stable/updates main contrib non-fr
               deb http://ftp.debian.org/debian stable-updates main contrib non-free
               deb http://ftp.debian.org/debian wheezy-backports main contrib non-free
 
+[squeeze]
```
Comment (dktrkranz) [inline etc/debomatic/distributions]: This should be squeeze-lts

File: etc/debomatic/distributions [hunk 2]
```diff
@@ -38,6 +38,12 @@ extramirrors: deb http://security.debian.org/ stable/updates main contrib non-fr
...
+suite: squeeze
+mirror: http://ftp.debian.org/debian
+components: main contrib non-free
+extramirrors: deb http://security.debian.org/ squeeze/updates main contrib non-free
```
Comment (dktrkranz) [inline etc/debomatic/distributions]: No longer needed (see https://wiki.debian.org/LTS/Using)

File: etc/debomatic/distributions [hunk 3]
```diff
@@ -38,6 +38,12 @@ extramirrors: deb http://security.debian.org/ stable/updates main contrib non-fr
...
+              deb http://ftp.debian.org/debian squeeze-updates main contrib non-free
```
Comment (dktrkranz) [inline etc/debomatic/distributions]: s/updates/lts/

**Comment by mfvescovi** [comment]
Hopefully fixed.

**Pull request merged**